packages/server/src/db/records/badge.ts

```diff
@@ -79,7 +75,8 @@ const insertFn =
       creator: uri.host,
       subject: obj.subject,
       assertionType: obj.assertion.type,
-      assertionTag: (obj.assertion as Badge.AppBskyBadgeTagAssertion).tag,
+      assertionTag:
+        (obj.assertion as Badge.AppBskyBadgeTagAssertion).tag || null,
```

Comment:
Despite the typescript type, this could technically be any of the badge assertion types, does that sound right?

Reply:
yup. this was just TS giving me hell on the type 😒